### DISCUSSION TITLE
Add GetState method to get the state of th entity

### DISCUSSION THREAD

**Discussion by pouliotf** [ffernandolima/ef-core-data-access#23] [Ideas]
There is a SetState on the Repository to change the state of the entity, but I think the same most be offer to getting the state of the entity.

**Comment by ffernandolima** [comment]
Hi!

It was already addressed by this [PR](https://github.com/ffernandolima/ef-core-data-access/pull/24) and released a new library version (7.0.11).

Thanks.
